app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -239,6 +299,21 @@ export default class UserMenu extends Component {
     ];
   }
 
+  get _assignedNotificationTypes() {
```

Comment:
I'd move this getter into `_unassignedNotificationTypes` below, see the comment on `_unassignedNotificationTypes` for why...